app/views/layouts/application.html.erb

```diff
@@ -88,6 +88,9 @@
         <% if lookup_context.template_exists?("footer", "layouts", true) %>
           <%= render :partial => "layouts/footer" %>
         <% else %>
+          <% if @user && @user.is_admin? %>
```

Comment:
Please remove this and just note the link in `README.md`. Tags are edited so infrequently that it's not worth the space. It can get a link on the admin dashboard when that gets built.

Reply:
I was thinking about this, and the page I used for it is a nice screen of all tags sized by popularity that's been in the app for a while. Oddly enough, it doesn't seem to be linked anywhere in the app that I've seen. What if we left the link to that page there for all users instead of restricting to admin? Just a thought, deleting it is fine if we don't want to do that.

Reply:
Yeah, let's just delete.